src/Entity/Redirect.php

```diff
@@ -65,7 +65,7 @@ class Redirect extends ContentEntityBase {
    */
   public static function generateHash($source_path, array $source_query, $language) {
     $hash = array(
-      'source' => $source_path,
+      'source' => strtolower($source_path),
```

Comment:
That would be great, you can also just share a gist or so, not sure if we want to start a precedent by adding an upgrade path function already now :)